omniduct/databases/_schemas.py

```diff
@@ -87,7 +87,7 @@ class TableDesc(Table):
     def desc(self):
         """pandas.DataFrame: The description of this SQL table."""
         return pd.DataFrame(
-            [[col.name, col.type] for col in self.columns.values()],
+            [[col.name, col.type.compile(self.bind.dialect)] for col in self.columns.values()],
```

Comment:
the generic dialect compiler doesn't always know how to compile dialect specific types

Reply:
Asking to learn: What does that do exactly? It's been a long time since I've played with the internals of SQLAlchemy.

It seems to work nicely for me in testing, but I'm not sure how this modifies the behaviour in practice.

Reply:
Ah... I see. We previously returned the SQLAlchemy type, whereas this returns a string tailored to the dialect being used. This looks good to me.